nym-vpn-core/crates/nym-vpnc/Cargo.toml

```diff
@@ -39,6 +39,11 @@ vergen = { workspace = true, default-features = false, features = [
     "cargo",
 ] }
 
+[target.'cfg(windows)'.dependencies]
```

Comment:
Neither of these crates seem to be used in `nym-vpnc` based on a quick search with Github